tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/JMSMessageProducerTest.java

```diff
@@ -70,10 +70,31 @@ public void testAnonymousProducer() throws Exception {
    }
 
    @Test(timeout = 30000)
-   public void testAnonymousProducerWithAutoCreation() throws Exception {
-      Connection connection = createConnection();
+   public void testAnonymousProducerWithQueueAutoCreation() throws Exception {
+      try (Connection connection = createConnection()) {
+         Session session = connection.createSession(false, Session.AUTO_ACKNOWLEDGE);
+         Queue queue = session.createQueue(UUID.randomUUID().toString());
+         MessageProducer p = session.createProducer(null);
 
-      try {
+         TextMessage message = session.createTextMessage();
+         message.setText("hello");
+         // this will auto-create the address
+         p.send(queue, message);
```

Comment:
How so? I do need at least one send when dealing with anonymous producer. That is the point of the test, to send a message on the broker, that is the point where any auto-creation happens.

Reply:
I believe it is the second send you don't need, the first on create the address which result in a MULTICAST address being created which sets up the scene for the create consumer to fail because the consumer is creating a JMS Queue and will get clobbered because that address is already MULTICAST or something.

Reply:
Thanks, I get it now. Ok, I don't need it.

Reply:
Yeah, thats what I meant to say, the second send.